src/app/thankYou/thankYou.component.js

```diff
@@ -43,6 +49,14 @@ class ThankYouController{
         if(this.purchase.paymentMeans.self.type === 'elasticpath.purchases.purchase.paymentmeans'){ //only credit card type has billing address
           this.billingAddress = this.orderService.formatAddressForTemplate(this.purchase.paymentMeans['billing-address'].address);
         }
+
+        // Display Account Benefits Modal when GUEST checkout
```

Comment:
Alright... Huh. If that's the case do we need the public check on the next line? (I probably shouldn't have started this thread on a comment...)